internal/dataplane/parser/parser.go

```diff
@@ -124,7 +124,7 @@ func shouldEnableParserExpressionRoutes(
 
 // LicenseGetter is an interface for getting the Kong Enterprise license.
 type LicenseGetter interface {
-	GetLicense() kong.License
+	GetLicense() (kong.License, bool)
```

Comment:
The meaning of the bool isn't immediately clear from the signature alone, so:

```suggestion
	// GetLicense() returns a license (if one is available) and a boolean indicating if a license is available.
	GetLicense() (kong.License, bool)
```